asv/benchmarks/simulation/bench_contacts.py

```
@@ -145,6 +248,54 @@ def time_simulate(self):
         wp.synchronize_device()
 
 
+class FastConvexCollision:
+    """Benchmark lean-hull and mixed-type convex collision workloads."""
+
+    params = (CONVEX_COLLISION_CASES,)
+    param_names = ["case"]
```

Ruff (0.16.1)

[warning] Mutable default value for class attribute

(RUF012)